lib/sport_ngin_aws_auditor/ec2_instance.rb

```diff
@@ -52,14 +62,17 @@ def get_more_info(instances, client)
       private :get_more_info
     end
 
-    attr_accessor :id, :name, :platform, :availability_zone, :scope, :instance_type, :count, :stack_name, :tag_value, :tag_reason, :expiration_date, :count_remaining
-    def initialize(ec2_instance, tag_name, count=1)
+    attr_accessor :id, :name, :platform, :availability_zone, :scope, :instance_type, :count, :stack_name, :tag_value, :tag_reason, :expiration_date, :count_remaining, :ec2_classic_support
```

Comment:
http://sportngin.github.io/styleguide/ruby.html#source-code-layout_L17

You should change this super long single line into a more readable multi-line.